browser/brave_ads/ads_stats_updater_helper.cc

```diff
@@ -0,0 +1,77 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_ads/ads_stats_updater_helper.h"
+
+#include "base/bind.h"
+#include "brave/components/brave_ads/common/pref_names.h"
+#include "chrome/browser/browser_process.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/common/pref_names.h"
+#include "components/pref_registry/pref_registry_syncable.h"
+#include "components/prefs/pref_service.h"
+
+namespace brave_ads {
+
+AdsStatsUpdaterHelper::AdsStatsUpdaterHelper()
```

Comment:
We don't need to prepend Ads because we are in the brave_ads namespace, I.e we could just have brave_ads::StatsUodaterHelper

Reply:
I see. Was just following the convention of other files in that dir. will rename

Reply:
ah yes we are gradually removing the verbosity where it is not needed

Reply:
ok noted, will keep this in mind going forward

Reply:
as mentioned in DM we do however need StatsUpdater -> BraveStatsUpdater to be consistent with other references in the code